compose/ui/ui/samples/build.gradle

```diff
@@ -25,17 +25,17 @@ plugins {
 }
 
 dependencies {
-    kotlinPlugin(project(":compose:compiler:compiler"))
+    kotlinPlugin(projectOrArtifact(":compose:compiler:compiler"))
 
     implementation(libs.kotlinStdlib)
 
-    compileOnly(project(":annotation:annotation-sampled"))
+    compileOnly(projectOrArtifact(":annotation:annotation-sampled"))
 
-    implementation(project(":compose:animation:animation-core"))
+    implementation(projectOrArtifact(":compose:animation:animation-core"))
     implementation("androidx.compose.foundation:foundation-layout:1.0.0")
     implementation("androidx.compose.material:material:1.0.0")
-    implementation(project(":compose:runtime:runtime"))
-    implementation(project(":compose:ui:ui"))
+    implementation(projectOrArtifact(":compose:runtime:runtime"))
+    implementation(projectOrArtifact(":compose:ui:ui"))
```

Comment:
Can we leave the projects included in the playground as project dependencies so we don't accidentally replace them with snapshots if we forget to include them?

Reply:
Swapped `projectOrArtifact(":compose:ui:ui")` with `project(...)`.

`:compose:runtime:runtime` is not being loaded at all, so that will need to remain `projectOrArtifact`.

Reply:
Given that is super common for these projects to depend on project("compose:runtime:runtime"), should we pull it in?

Reply:
Pulling in `:compose:runtime:runtime`.